webapp/sass/responsive/_mobile.scss

```diff
@@ -209,11 +209,12 @@
 
         .post__header {
             margin-bottom: 0;
+            padding-right: 50px;
```

Comment:
@thomchop I think you've added some extra padding you the post header that is not required (you probably added it considering how the flag icon is floating, which is fine). I would atleast prefer the header to appear in the same line for an iphone 6 resolution. Based on the max-width of the popover, I think if we just reduce the padding for the `post__header` to `40px`, that would work fine.